src/ripper/ripcddialog.h

```diff
@@ -83,5 +85,6 @@ class RipCDDialog : public QDialog {
   Ripper* ripper_;
   bool working_;
   CddaSongLoader* loader_;
+  QBasicTimer media_changed_timer_;
```

Comment:
Why not use `QTimer`? It's more consistent with what we use elsewhere.

Reply:
I based this on the implementation of progress polling in `TranscodeDialog`, which uses the `QBasicTimer`. Seems then like I have picked the one example that's inconsistent with the rest of the code. I can of course change this to `QTimer` here, but shouldn't we then change the `TranscodeDialog` as well then?

Reply:
I replace the `QBasicTimer` with a `QTimer` now. Should I look into doing the same in `TranscodeDialog` then @hatstand ?